lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -120,7 +120,7 @@ function Test-ProjectPackage {
   param ($Path, $Algorithm = 'SHA256', $Hash)
   if (!$env:Valid_ProjectPackage){
     Write-Verbose "Testing the $Algorithm hash for $path."
-    $ActualHash = if (Is-FIPS) {
+    $ActualHash = if ($(Is-FIPS) -ge 1 -or $(Get-PowershellVersion) -le 3) {
```

Comment:
I'd doubt that produces a perf improvement `Get-FileHash` calls `dispose` on its objects as well. See `Get-Command Get-FileHash | fl`. If there is no improvement in perf (always worth casual benchmarking) I'd consolidate to one implementation.